lib/toShahmukhi.js

```diff
@@ -1,48 +1,72 @@
-const escapeStringRegexp = require( 'escape-string-regexp' )
-
 const vishraams = Object.values( require( './vishraams' ) )
 
+export default function escapeUnicodeStringRegexp(string) {
```

Comment:
Why the export default here?

Reply:
No reason to have that there actually, initially I had just used the same function from the escape regexp library to just escape the unicode characters and nothing else, but there isn't even a need to have a separate function for that really. I've removed the function and just appended the unicode escape where it's needed.